gql_http_link/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: gql_http_link
 author: Klāvs Priedītis <klavs@prieditis.lv>
-version: 0.2.5
+version: 0.3.0
```

This is not a breaking change. It can be 0.2.6
```suggestion
version: 0.2.6
```